splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/upgrade/SpliceCatalogUpgradeScripts.java

```diff
@@ -81,7 +81,7 @@ public SpliceCatalogUpgradeScripts(SpliceDataDictionary sdd, TransactionControll
 
         scripts = new ArrayList<>();
         addUpgradeScript(baseVersion4, 2020, new UpgradeAddConglomerateNumberIndex(sdd, tc));
-        addUpgradeScript(baseVersion4, 2024, new UpgradeScriptToPrioritizeSchemaIdInSystemIndices(sdd, tc));
+        addUpgradeScript(baseVersion4, 2024, new UpgradeScriptToPrioritizeSchemaIdInSystemIndices(sdd, tc, startParams));
         // DB-11296: UpgradeConglomerateTable has to be executed first, because it adds a system table
```

Comment:
for new build has to be changed to 2025

Reply:
It should still be 2024 in theory, because the index were changed since 2024, but we should recommend customer install 2025 build to trigger upgrade.